gocd/templates/bash/deploy.sh

```diff
@@ -7,47 +7,31 @@ eval $(regions-project-env-vars --region="${SENTRY_REGION}")
   --label-selector="${LABEL_SELECTOR}" \
   --image="us-docker.pkg.dev/sentryio/snuba-mr/image:${GO_REVISION_SNUBA_REPO}" \
   --container-name="api" \
-  --container-name="consumer" \
   --container-name="dlq-consumer" \
-  --container-name="eap-items-consumer" \
   --container-name="eap-items-subscriptions-executor" \
   --container-name="eap-items-subscriptions-scheduler" \
   --container-name="errors-replacer" \
   --container-name="events-subscriptions-executor" \
   --container-name="events-subscriptions-scheduler" \
-  --container-name="generic-metrics-counters-consumer" \
-  --container-name="generic-metrics-distributions-consumer" \
-  --container-name="generic-metrics-sets-consumer" \
   --container-name="genmetrics-counters-subs-scheduler" \
   --container-name="genmetrics-counts-subs-executor" \
   --container-name="genmetrics-distributions-subs-executor" \
   --container-name="genmetrics-distributions-subs-scheduler" \
   --container-name="genmetrics-sets-subs-executor" \
   --container-name="genmetrics-sets-subs-scheduler" \
-  --container-name="group-attributes-consumer" \
```

Comment:
https://github.com/getsentry/ops/blob/68b5a2c1d09dc53a4b21ba64d2398e412114a271/k8s/services/snuba/_values.yaml#L432-L438

group attributes is not a rust consumer